packages/authgear-capacitor/android/src/main/java/com/authgear/capacitor/Authgear.java

```diff
@@ -301,20 +307,82 @@ void removeBiometricPrivateKey(String alias) throws Exception {
         keyStore.deleteEntry(alias);
     }
 
+    void createDPoPPrivateKey(String kid) throws Exception {
+        if (Build.VERSION.SDK_INT < Build.VERSION_CODES.M) {
+            return;
```

Comment:
It return silently because we don't want device that does not support dpop to be unable to use the SDK. i.e. Unsupported device should be no-op when using DPoP functions.

Reply:
I think we can add one more function in the native module, `checkDPoPAvailable(): boolean`. And the js code will call this function to determine if other dpop functions can be called. In this way, we can throw error here.

Reply:
> I think we can add one more function in the native module, checkDPoPAvailable(): boolean. And the js code will call this function to determine if other dpop functions can be called. In this way, we can throw error here.

Yes, exactly. Please do so.